src/session10/livepractice/Animal.java

```diff
@@ -0,0 +1,50 @@
+package session10.livepractice;
+
+public class Animal {
+
+    private String name;
+
+    public Animal(String name) {
+        this.name = name;
+    }
+
+    public void eat(){
```

Comment:
Please don't forget to format the code.

![image](https://github.com/Adrianpush/IT-School/assets/31310899/e83b064e-5487-430b-af33-45c20ab1cf12)